### DISCUSSION TITLE
Visual distinction between internal and external links

### DISCUSSION THREAD

**Discussion by cstby** [outline/outline#6774] [Ideas]
# Customer Need

Users want to know where a hyperlink will take them. Moreover, users have come to expect this link hinting because of its ubiquity among competitors (including Google Docs, Notion, and Obsidian).

# Possible ways to meet this need

- Add the website favicon to external links.
- Add an icon (such as the outline icon or workspace icon) to internal links.
- Add an icon (such as the chain hyperlink icon) to external links.
- Change visual treatment of internal links to a pill (similar to Google Docs).

**Comment by tommoor** [comment]
👍 will definitely get to this soon

**Reply by mcdamo** [reply]
Very useful to have visual distinction between internal and external links.

Wasn't something like this implemented before? https://github.com/outline/outline/pull/3100

It was mentioned in the changelog https://www.getoutline.com/changelog/fit-and-finish
> External links now show an icon

**Reply by tommoor** [reply]
Yes, when you use `@` mention to insert a document - this is the best way.
